src/http/client.cr

```diff
@@ -380,94 +380,96 @@ class HTTP::Client
     end
   {% end %}
 
-  # Executes a POST with form data. The "Content-type" header is set
-  # to "application/x-www-form-urlencoded".
-  #
-  # ```
-  # client = HTTP::Client.new "www.example.com"
-  # response = client.post_form "/", "foo=bar"
-  # ```
-  def post_form(path, form : String | IO, headers : HTTP::Headers? = nil) : HTTP::Client::Response
-    request = new_request("POST", path, headers, form)
-    request.headers["Content-type"] = "application/x-www-form-urlencoded"
-    exec request
-  end
+  {% for http_method in %w(post put patch) %}
+    # Executes a {{http_method.id.upcase}} with form data. The "Content-type" header is set
```

Comment:
A) Maybe add "request" after the method type? => `Executes a POST request with form data...`